<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextChatWidget</class>
 <widget class="QWidget" name="TextChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Aspia Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#TextChatWindow {
background: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="list_messages">
        <property name="styleSheet">
         <string notr="true">#list_messages {
border: 0;
}</string>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_status">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#label_status {
color: gray;
font-style: italic;
font-size: 10px;
background: white;
border-top: 0;
border-left: 0;
border-right: 0;
border-bottom: 1px solid #CDCDCD;
padding: 2px;
}</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="edit_message">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#edit_message {
border: 1px solid #CDCDCD;
border-radius: 3px;
background: #FAFAFA;
color: #505050;
padding: 4px;
}</string>
       </property>
       <property name="maxLength">
        <number>2048</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QToolButton" name="button_send">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Send message</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
color: #505050;
font: bold 11px;
background-color: #E9E9E9;
border: 1px solid #CDCDCD;
border-radius: 3px;
padding: 2px 2px 2px 2px;
}
QToolButton:hover {
background: #EBEBEB;
border: 1px solid #AAAAAA;
border-radius: 2px;
}
QToolButton:pressed {
background: #D7D7D7;
border: 1px solid #AAAAAA;
border-radius: 2px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/common.qrc">
             <normaloff>:/img/send.svg</normaloff>:/img/send.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="button_tools">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Tools</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
color: #505050;
font: bold 11px;
background-color: #E9E9E9;
border: 1px solid #CDCDCD;
border-radius: 3px;
padding: 2px 2px 2px 2px;
}
QToolButton:hover {
background: #EBEBEB;
border: 1px solid #AAAAAA;
border-radius: 2px;
}
QToolButton:pressed {
background: #D7D7D7;
border: 1px solid #AAAAAA;
border-radius: 2px;
}</string>
           </property>
           <property name="text">
            <string>···</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/common.qrc"/>
 </resources>
 <connections/>
</ui>
